pages/Nix/Hyprland on NixOS.md

```diff
@@ -146,3 +146,13 @@ in {
 
 If your themes for mouse cursors, icons or windows don't load correctly, see the
 relevant section in [Hyprland on Home Manager](../Hyprland-on-Home-Manager).
+
+If you prefer not to use Home Manager, you can also resolve the issues with GTK themes using dconf like so:
+
+\`\`\`ini
```

Comment:
I meant without the actual `\`

Reply:
I pushed resolve I think that automatically adds your changes, sorry if it does not, I'm not very familiar with github.